### ISSUE TITLE
Error while running demo.py

### ISSUE THREAD

**Issue by MritulaC** [DenisTome/Lifting-from-the-Deep-release#36]
Hi,
 Thanks for the wonderful application. I tried running demo.py and im getting the below mentioned error. I have installed Tensorflow1.12.0 and Opencv. Do let me know how to overcome this.

Thanks,
 Mritula

Error:
(Lift_Deep) E:\Lift_From_Deep\applications>python demo.py
Traceback (most recent call last):
  File "demo.py", line 11, in <module>
    from lifting import PoseEstimator
  File "E:\Lift_From_Deep/packages\lifting\__init__.py", line 1, in <module>
    from ._pose_estimator import *
  File "E:\Lift_From_Deep/packages\lifting\_pose_estimator.py", line 7, in <module>
    from . import utils
  File "E:\Lift_From_Deep/packages\lifting\utils\__init__.py", line 9, in <module>
    from .cpm import *
  File "E:\Lift_From_Deep/packages\lifting\utils\cpm.py", line 5, in <module>
    import tensorflow as tf
  File "C:\Users\user\Anaconda3\envs\Lift_Deep\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "C:\Users\user\Anaconda3\envs\Lift_Deep\lib\site-packages\tensorflow\python\__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\user\Anaconda3\envs\Lift_Deep\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\user\Anaconda3\envs\Lift_Deep\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\user\Anaconda3\envs\Lift_Deep\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\user\Anaconda3\envs\Lift_Deep\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 20, in swig_import_helper
    import _pywrap_tensorflow_internal
ImportError: No module named _pywrap_tensorflow_internal

Failed to load the native TensorFlow runtime.
See https://www.tensorflow.org/install/errors

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.

**Comment by MritulaC** [comment]
Sorry I think its my mistake. Tensorflow installation says that "TensorFlow is not supported on Windows with Python 2.7". Can I use Python higher version for this application?

**Comment by MritulaC** [comment]
Sorry I think its my mistake. Tensorflow installation says that "TensorFlow is not supported on Windows with Python 2.7". Can I use Python higher version for this application?

**Comment by DenisTome** [comment]
I think it shouldn't be problematic. You might need to change very few lines to adapt from 2.7 to 3+ but it should require few changes.

Let me know if you manage to do that, otherwise I'll have a look

**Comment by MritulaC** [comment]
As of now I found the issue was with setup.sh didnt run properly and I did the below. The models now uploaded and the demo worked. Now I am trying to use on my images. But not sure which part of the code to change or supply the input path, checking on it. Thanks

Downloaded the  *.tar.tz files manually by pasting the URLs from the setup.sh files and executed each instruction line by line from git bash.
